RELEASE-NOTE.md

```diff
@@ -0,0 +1,39 @@
+ceph-ansible stable-7.0 release note
+====================================
+
+Ansible version support
+-----------------------
+
+This version supports ansible-core 2.12
+
+Ceph version support
+--------------------
+
+This version can deploy Ceph Quincy.
+Note that the general intent for Ceph Ansible is to support one Ceph release only.
+The next version will support Reef, then the one after that will support S Ceph release.
+
+Added features
+----------------
+
+* Add RHCS 6 support.
+
+Removed features
+----------------
+
+Deprecation warnings and incoming removal
+-----------------------------------------
+
+*  use the ceph/ceph container image instead of ceph/daemon
```

Comment:
```suggestion
* The next release of ceph-ansible `stable-8.0` will drop `ceph/daemon` images support.
Use the ceph/ceph container image instead of ceph/daemon
```